### ISSUE TITLE
a11y: consider including both name and abbr to the title

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-avatar#46]
From https://github.com/vaadin/vaadin-avatar/issues/10#issuecomment-656802750

> Another improvement could be, to add the `abbr` if present inside title of `<vaadin-avatar>`.

> Examples:
> - name = Max Mustermann, abr = null -> title = Max Mustermann 
> - name = Max Mustermann, abr = MM -> title = Max Mustermann (MM)
> - name = null abr = MM -> title = MM
> - name = null abr = null -> title = anonymous (<- this should be customizable (i18n))

Currently, the second example (both name and abbr) is not supported. We can consider supporting it.
